core/amber/src/main/scala/Engine/SchemaSupport/constants/DataConstants.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * 
+ */
+package Engine.SchemaSupport.constants;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Zuozhi Wang (zuozhi)
+ */
+public class DataConstants {
```

Comment:
I saw that most of these files under constant folder are not being used anywhere. Pls remove the files/classes which we are not using

Reply:
done